operator/builtin/input/file/file_test.go

```diff
@@ -1116,6 +1180,20 @@ func waitForOne(t *testing.T, c chan *entry.Entry) *entry.Entry {
 	}
 }
 
+func waitForN(t *testing.T, c chan *entry.Entry, n int) []string {
+	messages := make([]string, 0, n)
+	for i := 0; i < n; i++ {
+		select {
+		case e := <-c:
+			messages = append(messages, e.Record.(string))
+		case <-time.After(time.Minute):
```

Comment:
Any reason for such a long timeout?

Reply:
Not in particular. Just copied from the method above. I agree it's too long though. I'll shorten both.